src/components/Options/index.tsx

```diff
@@ -76,39 +70,18 @@ interface Props {
 
 interface State {
   encoderSupportMap?: EncoderSupportMap;
-  showLoadingState: boolean;
 }
 
-const loadingReactionDelay = 500;
-
 export default class Options extends Component<Props, State> {
   state: State = {
     encoderSupportMap: undefined,
-    showLoadingState: false,
   };
 
-  /** The timeout ID between entering the loading state, and changing UI */
-  private loadingTimeoutId: number = 0;
-
   constructor() {
     super();
     encodersSupported.then(encoderSupportMap => this.setState({ encoderSupportMap }));
   }
 
-  componentDidUpdate(prevProps: Props, prevState: State) {
-    if (prevProps.loading && !this.props.loading) {
-      // Just stopped loading
-      clearTimeout(this.loadingTimeoutId);
-      this.setState({ showLoadingState: false });
-    } else if (!prevProps.loading && this.props.loading) {
-      // Just started loading
-      this.loadingTimeoutId = self.setTimeout(
-        () => this.setState({ showLoadingState: true }),
-        loadingReactionDelay,
-      );
-    }
-  }
```

Comment:
A lot of this stuff was needed for the "results" view (as in "154kb 91% smaller") which I've moved into its own component.